### PR TITLE
New release

### PR DESCRIPTION
**Pull request by nomadeel**
This PR takes the CHANGES files from the relevant repositories that have new releases and throws them into the docsite. Also update the configuration to point to the new releases.

### CODE REVIEW

**Comment by nomadeel** [comment]
I will update the discourse link to point to the new discourse release announcement once the release is finalised.

File: content_collections/_releases/sel4/12.1.0.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+* Changed some variables to use `BOOT_BSS` instead of `BOOT_DATA` to save space in the ELF file.
+* Replaced the `capDL()` function with a generic `debug_capDL` function that is intended to be implemented by all
+  architectures.
+* Reduced `printf`s stack usage.
+* Fixed `ksnprintf()` corner case handling.
+* Fixed NULL `printf` output wrapper handling.
+* Cleaned up the printing API implementation.
+* Changed code to pass buffer to `printf` output channel.
+* Refactored the kernel console handling.
+* Added support for `PRIu64 and `SEL4_PRIu_word` in the kernel.
```
Comment (mbrcknl) [inline content_collections/_releases/sel4/12.1.0.md:53]:
```suggestion
* Added support for `PRIu64` and `SEL4_PRIu_word` in the kernel.
```

Comment (nomadeel) [reply content_collections/_releases/sel4/12.1.0.md:53]: Good catch

**Review by lsf37 — APPROVED** [review]
I'll approve this so you don't have to wait when the others are done, but basically all text tweaks for the other PRs you'll also need to include in this one.

**Comment by nomadeel** [comment]
Yep, done.

**Pull request merged**